modules/javafx.graphics/src/main/native-glass/win/ViewContainer.cpp

```diff
@@ -547,8 +547,8 @@ void ViewContainer::HandleViewDeadKeyEvent(HWND hwnd, UINT msg, WPARAM wParam, L
     }
 
     // Since we handle dead keys ourselves, reset the keyboard dead key status (if any)
-    static BYTE kbState[256];
-    ::GetKeyboardState(kbState);
+    static BYTE kbState[256] = {};
```

Comment:
my c++ is a bit rusty, do we need `static` here?

Reply:
There's no point in this being static. I've updated the PR.